docs/content/content-blocks.md

```diff
@@ -66,6 +23,16 @@ Results in the following output:
 Here is a note!
 ```
 
+````{margin} A note on nesting
```

Comment:
perhaps margins/sidenotes should be `::::` as well?

Reply:
Way ahead of you 😄 https://github.com/executablebooks/MyST-Parser/issues/213
As I mentioned to @mmcky (https://github.com/executablebooks/jupyter-book/pull/911#discussion_r478763053):

the `:::` blocks are basically a new "breed" of Markdown specific directives (i.e. they require as input Markdown-It Tokens, rather than raw text).

As I plan to add more (see https://github.com/executablebooks/MyST-Parser/issues/213), I will try to formalise their usage, so that they are more modular and extensible
(currently they are a bit hard-coded into the renderer https://github.com/executablebooks/MyST-Parser/blob/bf7fd80c3273a4fea4be595bfb37a5b51b452760/myst_parser/docutils_renderer.py#L676)